### PR TITLE
Fix admin schedules and some bugs

### PR DESCRIPTION
**Pull request by adam-hart**
Schedules were totally broken on Admin, so I fixed them!  They still need some styling (cough cough @roseleaf), but they're functional at least.

@zendesk/volunteer

### CODE REVIEW

File: components/ServiceEdit/ServiceEdit.js [hunk 1]
```diff
@@ -3,10 +3,36 @@ import s from './ServiceEdit.css'
...
+const scheduleSorter = {
+  "-":        -1,
+  "Sunday":    0,
+  "Monday":    10000,
+  "Tuesday":   20000,
+  "Wednesday": 30000,
+  "Thursday":  40000,
+  "Friday":    50000,
+  "Saturday":  60000
+}
```
Comment (craig-day) [inline components/ServiceEdit/ServiceEdit.js]: why the randomly high numbers? Why not just 0-6?

File: components/ServiceEdit/ServiceEdit.js [hunk 2]
```diff
@@ -71,17 +97,47 @@ class ServiceEdit extends Component {
...
 
-  handleTimeChange = (event, start_or_end, metadata) => {
+  handleTimeChange = (event, value_changed, metadata) => {
     const { schedules } = this.props.service
     const newSchedules = schedules
 
-    if (start_or_end == 'start') {
-      newSchedules[metadata.scheduleNum][metadata.day]['opens_at'] = event.target.value.replace(':', '')
+    if (value_changed == 'start') {
+      newSchedules[metadata.scheduleNum]['opens_at'] = event.target.value.replace(':', '')
```
Comment (craig-day) [inline components/ServiceEdit/ServiceEdit.js]: this should be camelCase, not snake_case, `opensAt`

File: components/ServiceEdit/ServiceEdit.js [hunk 3]
```diff
@@ -71,17 +97,47 @@ class ServiceEdit extends Component {
...
+    } else if (value_changed == 'end') {
+      newSchedules[metadata.scheduleNum]['closes_at'] = event.target.value.replace(':', '')
```
Comment (craig-day) [inline components/ServiceEdit/ServiceEdit.js]: same as above

File: components/ServiceEdit/ServiceEdit.js [hunk 4]
```diff
@@ -159,6 +203,28 @@ class ServiceEdit extends Component {
...
+            className={s.addToSubsection}
+            onClick={this.newSchedule}
+            title={`Click to add a new schedule`}>
+            + Add
+          </button>
+        </div>
+        <div className={s.inputBox}>
+          <div className={s.inputGroup, s.row}>
+            <span className={s.inputLabel}>Schedule: </span>
+            {(this.sortSchedules(service.schedules || [])).map((schedule, index) => (
```
Comment (craig-day) [inline components/ServiceEdit/ServiceEdit.js]: shouldn't need to `|| []` anymore after merging #343

File: components/TimeRangePicker/TimeRangePicker.css [hunk 5]
```diff
@@ -16,3 +16,13 @@
   box-shadow: inset 0 1px 2px #ddd;
   color: #777;
 }
+
+.buttonStyle {
+  border: solid 1px #777;
+  cursor: pointer;
+  font-family: Helvetica,Arial,sans-serif;
```
Comment (craig-day) [inline components/TimeRangePicker/TimeRangePicker.css]: dont do this here, it should just be a global style

File: components/ServiceEdit/ServiceEdit.js (hunk 1 above)
Comment (adam-hart) [inline components/ServiceEdit/ServiceEdit.js]: So the existing times range 0-2359.  This was the easiest way I could think of to make sure that the previous day *always* weighted less than the next day.

Comment (thebritican) [inline components/ServiceEdit/ServiceEdit.js]: this is clever but we should make this more obvious later. but 🚢 it

**Pull request merged**